lib_test/test_common.ml

```diff
@@ -17,37 +17,24 @@
 open Git
 open Lwt.Infix
 
-let app_style = `Cyan
-let err_style = `Red
-let warn_style = `Yellow
-let info_style = `Blue
-let debug_style = `Green
-
 let pp ppf style h = Fmt.pf ppf "%a " Fmt.(styled style string) h
 
-let pp_header ppf (l, h) = match l with
-| Logs.App ->
-    begin match h with
-    | None -> ()
-    | Some h -> Fmt.pf ppf "[%a] " Fmt.(styled app_style string) h
-    end
-| Logs.Error -> pp ppf err_style (match h with None -> "ERROR" | Some h -> h)
-| Logs.Warning -> pp ppf warn_style (match h with None -> "WARN " | Some h -> h)
-| Logs.Info -> pp ppf info_style (match h with None -> "INFO " | Some h -> h)
-| Logs.Debug -> pp ppf debug_style (match h with None -> "DEBUG" | Some h -> h)
+let pad n x =
+  if String.length x > n then x else x ^ String.make (n - String.length x) ' '
```

Comment:
this doesn't seem to work. But maybe I'm doing something wrong... manual padding will go for now on.